src/readme/markdown.ts

```diff
@@ -0,0 +1,16 @@
+export const md = {
+  /**
+   * Create a markdown code block
+   *
+   * @param body body of the code block
+   * @param lang language to use eg `bash`
+   * @returns markdown formatted code block
+   */
+  code(lang: 'bash' | 'typescript', body: string): string {
```

Comment:
Why allow only these two languages?

Reply:
mostly to prevent the old `code(a,b)` vs `code(b,a)` problem as they would both be strings, other langs can be added as used.

Reply:
Would it be better to use an enum (or enum-like construct) for this?

Reply:
nope, enums are considered bad as they are not a standard, string unions are your best bet.

if the string union list gets long or needs to be reused, it could be extracted into a standalone type eg `export type MdCodeType = 'bash' | 'python' | 'typescript' | 'json'` but it is currently neither long or reused.